src/main/resources/en/aviation.yml

```diff
@@ -49,6 +49,6 @@ en:
arline.kz", "Sudan Airways",  "Sun Air of Scandinavia",  "Sun Country Airlines",  "Sun d'Or International Airlines", "SunExpress",  "Sunshine Express Airlines",  "Sunstate Airlines", "Sunwing Airlines",  "Superior Aviation",  "Surinam Airways",  "Swaziland Airlink", "Swift Air", "Swiftair", "Swiss European Air Lines", "Swiss International Air Lines", "Sylt Air", "Syrian Arab Airlines (Syrianair)", "TAAG Angola Airlines (Linhas Aéreas de Angola)", "TACA Peru", "TACV",  "TACV Cabo Verde Airlines", "TAF Linhas Aéreas", "Tajik Air", "TAM Airlines", "TAM Airlines (TAM Linhas Aereas)",  "TAME",  "Tampa Cargo", "Tandem Aero", "TAP Portugal",  "Tarom",  "Tassili Airlines",  "Tatarstan Airlines",  "Tavrey Airlines",  "Tbilaviamsheni",  "Thai AirAsia",  "Thai Airways International",  "Thomas Cook Airlines",  "Thomas Cook Airlines",  "Thomas Cook Airlines Scandinavia", "Thomson Airways", "Tiara Air",  "Tiger Airways",  "Tiger Airways Australia", "Titan Airways",  "TNT Airways",  "Tobruk Air",  "Toumaï Air Tchad", "Tradewinds Airlines", "Trans Air Benin", "Trans Air Congo", "Trans Maldivian Airways", "Trans Mediterranean Airlines",  "Trans States Airlines",  "Transaero Airlines",  "TransAsia Airways",  "TransAVIAexport Airlines", "Transcarga", "Transmile Air Services",  "Transwede Airways", "Transwest Air",  "Tri-MG Intra Asia Airlines",  "Trigana Air Service", "TRIP Linhas Aereas", "Triple Alpha Luftfahrtgesellschaft", "Tropic Air",  "TUIfly", "TUIfly Nordic",  "Tulpar Air Service", "Tunisair",  "TunisAir Express", "Turan Air",  "Turkish Airlines",  "Turkmenistan Airlines", "Twin Jet",  "Ukraine International Airlines",  "Ukrainian Cargo Airways",  "UM Airlines",  "Uni Air", "United Airlines",  "United Airways (Bangladesh)", "United Eagle Airlines", "United Parcel Service",  "Ural Airlines",  "US Airways",  "US Helicopter", "USA 3000 Airlines",  "USA Jet Airlines", "UTair Aviation",  "Uzbekistan Airways",  "Varig Logistica (VarigLog)", "Venezolana", "Viaggio Air",  "Vieques Air Link", "Vietnam Airlines",  "Viking Airlines", "VIM Airlines",  "Vintage Props and Jets", "VIP Ecuador", "Virgin America",  "Virgin Atlantic Airways",  "Virgin Blue",  "Virgin Nigeria Airways",  "Viva Macau",  "VivaAerobus", "Vladivostok Air",  "VLM Airlines",  "Volare Airlines",  "Volga-Dnepr Airlines",  "Voyageur Airways",  "VRG Linhas Aereas (VARIG)", "Vueling Airlines",  "WaltAir", "Wan Air", "Wasaya Airways", "WDL Aviation", "WebJet Linhas Aéreas", "Welcome Air",  "West Air Sweden",  "WestJet",  "Wideroe", "Wiggins Airways", "Wimbi Dira Airways", "Wind Jet",  "Wind Rose Aviation", "Windward Islands Airways", "Wings Air",  "Wings of Alaska",  "Wizz Air", "Wizz Air",  "World Airways",  "Wright Air Service",  "Xiamen Airlines",  "XL Airways France",  "XL Airways Germany", "Xtra Airways",  "Yakutia Airlines", "Yamal Airlines",  "Yangtze River Express",  "Yemenia - Yemen Airways", "Yeongnam Air", "Yuzmashavia",  "Zambia Skyways", "Zambian Airways",  "Zest Airways", "Zimex Aviation" ]
```

Comment:
There are a lot of places with 2 whitespaces instead of 1. Despite whitespaces, everything is fine!

Reply:
Yes. Can you please remove the double spaces here?

Reply:
Removed redundant spaces. Thank you for noticing that!